### ISSUE TITLE
Rename tablewares to singular.

### ISSUE THREAD

**Issue by oorestisime** [oorestisime/kayon-decor#41]
Create redirects for all tableware products https://nextjs.org/docs/pages/api-reference/next-config-js/redirects

**Comment by KostasTheodoro** [comment]
need to understand the point of what we want to do here, not sure i did

**Comment by artemis-toumazi** [comment]
Because all the other categories are singular (e.g. kitchenware, home furniture, home decor, etc.). So it must be tableware and not tablewareS.

**Comment by KostasTheodoro** [comment]
@artemis-toumazi yes so rename tablewares to tableware? i didn't get the redirect thing what we want them to redirect to, i'm confused maybe talk about this on the next meeting

**Comment by oorestisime** [comment]
If you change the slug of the category then we will have new links for all the products in that category right?
So we need to redirect all the "old" links to new links
